query-planner-js/src/QueryPlan.ts

```diff
@@ -26,6 +26,12 @@ export interface ParallelNode {
   nodes: PlanNode[];
 }
 
+export interface SubscriptionPlanNode {
+  kind: 'SubscriptionPlan';
```

Comment:
To be more consistent I suggest to just use `subscription`. What do you think ?

Reply:
When lookin at what we have here https://github.com/apollographql/router/blob/dev/apollo-router/src/query_planner/plan.rs#L65 nothing has a suffix but not a big deal, I will write an alias on our side to have consistency there